packages/frontend/src/components/Files/CategoryFiles.tsx

```diff
@@ -3,19 +3,49 @@ import { NodeItem } from './File';
 import { getRouterName } from '../../utils/helpers';
 import { ErrorMessage } from '../Notification/ErrorMessage';
 import { Spinner } from '../Spinner';
+import { useLayoutEffect, useState } from 'react';
+import { ButtonWrapper } from '../../styles/common';
+import { get } from 'lodash';
+import { useTranslation } from 'react-i18next';
 
 type TCategoryFilesProps = {
   categoryName: string;
 };
 
 export const CategoryFiles = ({ categoryName }: TCategoryFilesProps) => {
+  const { t } = useTranslation(['common']);
+  const [fileList, setFileList] = useState<any[]>([]);
```

Comment:
_Any_ chance to typify this any? Don't need to add all fields that the API gives. Just the ones we use/need.